raptorWeb/gameservers/models.py

```diff
@@ -70,7 +70,14 @@ class Server(models.Model):
 
     modpack_url = models.URLField(
         max_length=200, 
-        verbose_name="Link to Modpack")
+        verbose_name="Link to Modpack"
+    )
+
+    discord_announcement_channel_id = models.CharField(
+        max_length=200,
+        verbose_name="Discord Announcment Channel ID",
+        default="None"  
+    )
 
```

Comment:
Can create another CharField here for the Role ID associated with this server, rather than getting that list of ID's in the bot.